modelconverter/utils/onnx_tools.py

```
@@ -256,63 +254,189 @@ def __init__(
+        if from_modelproto:
+            onnx_model_1 = self._prev_onnx_model.SerializeToString()
+            onnx_model_2 = self._onnx_model.SerializeToString()
+        else:
+            onnx_model_1 = self.model_path.as_posix()
+            onnx_model_2 = self.output_path.as_posix()
+
+        ort_session_1 = ort.InferenceSession(onnx_model_1)
+        ort_session_2 = ort.InferenceSession(onnx_model_2)
+
+        inputs = {}
+        for input in ort_session_1.get_inputs():
```

Ruff (0.16.1)

[error] Variable `input` is shadowing a Python builtin

(A001)